lib/order.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'text'
+require_relative 'menu'
+
+class Order
+  include Menu
```

Comment:
This is legitimate. Orders should maybe know about their menus in this way, and it's conceivable that an order would also be a menu. Conceivable, but not too good. (I still think Menu should be a class)